ioctl/cmd/account/accountactions.go

```diff
@@ -0,0 +1,202 @@
+		ctx = metautils.NiceMD(jwtMD).ToOutgoing(ctx)
+	}
+
+	requestGetAccount := iotexapi.GetAccountRequest{
+		Address: addr,
+	}
+	accountResponse, err := cli.GetAccount(ctx, &requestGetAccount)
+	numActions := accountResponse.AccountMeta.GetNumActions()
+	fmt.Println("Total:", numActions)
+	requestGetAction := iotexapi.GetActionsRequest{
+		Lookup: &iotexapi.GetActionsRequest_ByAddr{
+			ByAddr: &iotexapi.GetActionsByAddressRequest{
+				Address: addr,
+				Start:   numActions - *countLimit - skip,
+				Count:   *countLimit,
+			},
+		},
+	}
+	response, err := cli.GetActions(ctx, &requestGetAction)
+	if err != nil {
+		sta, ok := status.FromError(err)
+		if ok {
+			return output.NewError(output.APIError, sta.Message(), nil)
+		}
+		return output.NewError(output.NetworkError, "failed to invoke GetActions api", err)
+	}
+	if len(response.ActionInfo) == 0 {
+		return output.NewError(output.APIError, "no action info returned", nil)
+	}
+	showFields := []interface{}{
+		"Hash",
+		//"Time",
+		"Status",
+		"Sender",
+		"Type",
+		"To",
+		"Contract",
+		"Amount",
+	}
+	tbl := table.New(showFields...)
+
+	for k := range response.ActionInfo {
+		k = len(response.ActionInfo) - 1 - k
+		actionInfo := response.ActionInfo[k]
+		requestGetReceipt := &iotexapi.GetReceiptByActionRequest{ActionHash: actionInfo.ActHash}
+		responseReceipt, err := cli.GetReceiptByAction(ctx, requestGetReceipt)
+		if err != nil {
+			sta, ok := status.FromError(err)
+			if ok {
+				fmt.Println(output.NewError(output.APIError, sta.Message(), nil))
+			} else {
+				fmt.Println(output.NewError(output.NetworkError, "failed to invoke GetReceiptByAction api", err))
+			}
+			continue
+		}
+		amount := "0"
+		transfer := actionInfo.Action.Core.GetTransfer()
+		if transfer != nil {
+			amount, _ = util.StringToIOTX(transfer.Amount)
+
+		}
+		tbl.AddRow(
+			actionInfo.ActHash,
+			//getActionTime(actionInfo),
+			iotextypes.ReceiptStatus_name[int32(responseReceipt.ReceiptInfo.Receipt.GetStatus())],
+			actionInfo.Sender,
+			getActionTypeString(actionInfo),
+			getActionTo(actionInfo),
+			getActionContract(responseReceipt),
+			amount+" IOTX",
+		)
+	}
+	tbl.Print()
+	return nil
+}
+
+func getActionContract(responseReceipt *iotexapi.GetReceiptByActionResponse) string {
+	contract := responseReceipt.ReceiptInfo.Receipt.GetContractAddress()
+	if contract == "" {
+		contract = "-"
+	}
+	return contract
+}
+
+func getActionTypeString(actionInfo *iotexapi.ActionInfo) string {
+	actionType := fmt.Sprintf("%T", actionInfo.Action.Core.GetAction())
+	return strings.TrimLeft(actionType, "*iotextypes.ActionCore_")
+}
+
+func getActionTo(actionInfo *iotexapi.ActionInfo) string {
+	recipient := ""
+	switch getActionTypeString(actionInfo) {
+	case "Transfer":
+		transfer := actionInfo.Action.Core.GetTransfer()
+		recipient = transfer.GetRecipient()
+	case "Execution":
+		transfer := actionInfo.Action.Core.GetExecution()
+		recipient = transfer.GetContract()
+	}
+	if recipient == "" {
```

Comment:
define it and init:
```
recipient := "_"
```
so we don't need this if{}?

Reply:
`recipient = transfer.GetContract()` can return empty string, need if{} judge